Add info on Tenderly (#1006)
Consensys/doc.linea: docs/get-started/tooling/security/tenderly.mdx

---
title: Tenderly

> description: Tenderly offers robust monitoring and security solutions for Linea blockchain with features including configurable alerts, automated Web3 Actions, and simulation capabilities to predict transaction outcomes before execution.

Reviewer: ```suggestion
description: Monitoring and security solutions including configurable alerts, automated Web3 Actions, and transaction simulation.
```

---

> Tenderly provides comprehensive monitoring and security solutions for Linea, allowing you to stay informed about on-chain events, and respond proactively to potential issues. 

Reviewer: ```suggestion
Tenderly provides comprehensive monitoring and security solutions for Linea, allowing you to stay informed about onchain events and respond proactively to potential issues. 
```

> - Configure [**Tenderly Alerts**](https://docs.tenderly.co/alerts/intro-to-alerts?mtm_campaign=ext-docs&mtm_kwd=linea) to get notifications on transactions and contract events, allowing you to stay aware of critical issues and to signal problems to external webhooks. Use the [Alert API](https://docs.tenderly.co/reference/api#/operations/createAlert?mtm_campaign=ext-docs&mtm_kwd=linea) for dynamic configuration and building more  powerful triggering criteria.

Reviewer: ```suggestion
- Configure [**Tenderly Alerts**](https://docs.tenderly.co/alerts/intro-to-alerts?mtm_campaign=ext-docs&mtm_kwd=linea) to get notifications on transactions and contract events, allowing you to stay aware of critical issues and to signal problems to external webhooks. Use the [Alert API](https://docs.tenderly.co/reference/api#/operations/createAlert?mtm_campaign=ext-docs&mtm_kwd=linea) for dynamic configuration and building more powerful triggering criteria.
```

- Use [**Web3 Actions**](https://docs.tenderly.co/web3-actions/intro-to-web3-actions?mtm_campaign=ext-docs&mtm_kwd=linea) to automate predefined responses, improving security and user experience. 

> - Leverage [**Simulation RPC**](https://docs.tenderly.co/simulations/single-simulations#simulate-via-rpc?mtm_campaign=ext-docs&mtm_kwd=linea) to predict transaction execution beforehand, and get insights about asset changes, state changes, and gas usage. 

Reviewer: ```suggestion
- Leverage [**Simulation RPC**](https://docs.tenderly.co/simulations/single-simulations#simulate-via-rpc?mtm_campaign=ext-docs&mtm_kwd=linea) to predict transaction execution, and get insights about asset changes, state changes, and gas usage. 
```

>   - It's possible to simulate single transactions with [`tenderly_simulateTransaction`](https://docs.tenderly.co/node/rpc-reference/linea/tenderly_simulateTransaction?mtm_campaign=ext-docs&mtm_kwd=linea) 

Reviewer: ```suggestion
  - You can simulate single transactions with [`tenderly_simulateTransaction`](https://docs.tenderly.co/node/rpc-reference/linea/tenderly_simulateTransaction?mtm_campaign=ext-docs&mtm_kwd=linea) 
```
